CHANGELOG.md

```diff
@@ -4,7 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/pusher/pusher-platform-node/compare/0.15.3...HEAD)
```

Comment:
It's not clear to me what these unreleased changes are. This was from a year ago so I assume it's safe to remove this line...?

Reply:
I think the idea is always to keep this one around and move it up every time you release.

Reply:
If there's nothing under the heading then there aren't any unreleased changes, it's just a placeholder for unreleased  changes that get merged in to master but not released. In theory your last change would have been listed here, but since you're releasing it immediately it's not a big deal. :) see https://keepachangelog.com/en/1.0.0/#effort

I would leave it in but change it to

```suggestion
## [Unreleased](https://github.com/pusher/pusher-platform-node/compare/0.15.5...HEAD)
```